Makefile

```
@@ -99,41 +102,46 @@ mocks:
 	counterfeiter -o ./core/database/mocks/database.i.go \
 		./core/database/database.i.go DBClient
 
-# Runs core service
+## core: Runs core service
 .PHONY: core
 core:
 	go run core/main.go run --dev \
 		--tls.cert dev/certs/127.0.0.1.crt \
 		--tls.key dev/certs/127.0.0.1.key $(FLAGS)
 
-# Runs API gateway
+## gateway: Runs API gateway
 .PHONY: gateway
 gateway:
 	go run gateway/main.go run --dev \
 		--core.cert dev/certs/127.0.0.1.crt $(FLAGS)
 
+## gateway-tls: Runs gateway tls 
 .PHONY: gateway-tls
 gateway-tls:
 	go run gateway/main.go run --dev \
 		--core.cert dev/certs/127.0.0.1.crt \
 		--tls.cert dev/certs/127.0.0.1.crt \
 		--tls.key dev/certs/127.0.0.1.key $(FLAGS)
 
-# Runs web app
+## web: Runs web app
 .PHONY: web
 web:
 	( cd frontend ; npm start )
 
-# Builds binary for pinpoint-core
+## pinpoint-core: Builds binary for pinpoint-core
 .PHONY: pinpoint-core
 pinpoint-core:
 	go build -o ./bin/pinpoint-core \
     -ldflags "-X main.Version=$(VERSION)" \
     ./core $(FLAGS)
 
-# Builds binary for pinpoint-gateway
+## pinpoint-gateway: Builds binary for pinpoint-gateway
 .PHONY: pinpoint-gateway
 pinpoint-gateway:
 	go build -o ./bin/pinpoint-gateway \
     -ldflags "-X main.Version=$(VERSION)" \
     ./gateway $(FLAGS)
+
```

```suggestion
.PHONY: help
```

we don't have a file called `help`, but... just in case